### PR TITLE
[ModelZoo] add models base image

### PR DESCRIPTION
**Pull request by typhoonzero**
1. build the base image: `cd base_image && docker build -t sqlflow/modelzoo_base .`
1. build sqlflow_models image: `cd sqlflow_models && docker build -t sqlflow/sqlflow_models .`
1. then the models can be imported by: docker run --rm -e PYTHONPATH=/ sqlflow_models python -c "from sqlflow_models import DNNClassifier"

TODO: Add tests using these docker images to ensure both SQLFlow and ElasticDL can use these model: https://github.com/sql-machine-learning/models/pull/25#discussion_r344876585

### CODE REVIEW

**Review by Yancey0623 — APPROVED** [review]
> then the models can be imported by: docker run --rm -e PYTHONPATH=/ sqlflow_models python -c "from sqlflow_models import DNNClassifier"

Just a tiny comment, maybe we can run the SQLFlow submitter script in the future: `docker run --rm -e PYTHONPATH=/ sqlflow_models python -m sqlflow_submitter.tensorflow.train {IR-JSON}`

**Comment by typhoonzero** [comment]
> > then the models can be imported by: docker run --rm -e PYTHONPATH=/ sqlflow_models python -c "from sqlflow_models import DNNClassifier"
> 
> Just a tiny comment, maybe we can run the SQLFlow submitter script in the future: `docker run --rm -e PYTHONPATH=/ sqlflow_models python -m sqlflow_submitter.tensorflow.train {IR-JSON}`

Sure, we will update to use the base image provided by Couler.

File: sqlflow_models/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM sqlflow/modelzoo_base
+
+RUN pip install tensorflow==2.0.0b1 scikit-learn==0.20.0 numpy==1.16.2 pandas==0.25.1
+ADD *.py /sqlflow_models/
```
Comment (tonyyang-svail) [inline sqlflow_models/Dockerfile:4]: why not `python setup.py install`? It should cover line 3 and line 4.

Comment (typhoonzero) [reply sqlflow_models/Dockerfile:4]: The future model zoo only require a Dockerfile for each model collection, then model developers don't need to care about how to write a `setup.py`.

**Pull request merged**